### PR TITLE
add xproto build dependency for makedepend v1.0.5 foss/2016a

### PR DESCRIPTION
**Pull request by SimonPinches**
(created using `eb --new-pr`)

I needed to add an xproto build dependency to get this to work since it requires xproto >= 7.0.17 and this wasn't met on our system.

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/m/makedepend/makedepend-1.0.5-foss-2016a.eb [hunk 1]
```diff
@@ -10,7 +10,11 @@ toolchain = {'name': 'foss', 'version': '2016a'}
 toolchainopts = {'optarch': True}
 
 sources = [SOURCE_TAR_GZ]
-source_urls = [XORG_UTIL_SOURCE]
+source_urls = ['http://xorg.freedesktop.org/releases/individual/util']
```
Comment (boegel) [inline easybuild/easyconfigs/m/makedepend/makedepend-1.0.5-foss-2016a.eb]: @SimonPinches what's wrong with using `XORG_UTIL_SOURCE`?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar03.gligar.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/43a8cd99ba0872ea9286171cacd2336b for a full test report.

File: easybuild/easyconfigs/m/makedepend/makedepend-1.0.5-foss-2016a.eb (hunk 1 above)
Comment (SimonPinches) [inline easybuild/easyconfigs/m/makedepend/makedepend-1.0.5-foss-2016a.eb]: I just started by using one of the other `makedepend` eb config files as a template and this is what was contained in there...

**Comment by boegel** [comment]
@SimonPinches https://github.com/SimonPinches/easybuild-easyconfigs/pull/5 makes this change consistent across the makedepend 1.0.5 easyconfigs

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2064.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/b345ebd9e15a4df83eed4e6e1fe878f0 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 25 out of 25 (6 easyconfigs in this PR)
node2401.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/1eb0a78597eec303e9051d96ff4b5423 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @SimonPinches!

**Pull request merged**